<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BullsAndCows</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="NewGame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Records">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>15</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Records</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Check">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>55</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterNum">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>55</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>55</y>
      <width>69</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter num: </string>
    </property>
   </widget>
   <widget class="QLabel" name="RezultLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>15</y>
      <width>170</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="ResultTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>380</width>
      <height>400</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>190</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Result</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
